__tests__/app.test.js

```diff
@@ -96,16 +96,18 @@ describe("GET /api/articles/:article_id", () => {
     return request(app)
       .get(`/api/articles/${testArticleId}`)
       .then((response) => {
-        const article = response.body.article[0];
+        const article = response.body;
         expect(response.status).toBe(200);
-        expect(article).toHaveProperty("article_id");
-        expect(article).toHaveProperty("title");
-        expect(article).toHaveProperty("topic");
-        expect(article).toHaveProperty("author");
-        expect(article).toHaveProperty("body");
-        expect(article).toHaveProperty("created_at");
-        expect(article).toHaveProperty("votes");
-        expect(article).toHaveProperty("article_img_url");
+        expect(article).toMatchObject({
+          article_id: testArticleId,
```

Comment:
We know the specific article which is being returned here (article 2), therefore we could also check we get the expected values back rather than just expect.any(String)